### ISSUE TITLE
let the user provide the file extension 

### ISSUE THREAD

**Issue by motazsaad** [deanmalmgren/textract#134]
Hi, 

It would be very nice if you provide the file extension. It is very useful for files with no extensions. 
So if the user provides just the file name, then the process function extracts the file extension, and if he provides the extension, then the process function consider this extension. 

Thanks
Motaz

**Comment by deanmalmgren** [comment]
Thanks for proposing @motazsaad! I'm completely open to a PR that offers this option.

As a workaround, I'd recommend renaming your extensionless file to something that has an appropriate extension prior to running it through textract. For example:

```sh
mv my-file my-file.pdf
textract my-file.pdf > my-file.txt
mv my-file.pdf my-file
```

**Comment by deanmalmgren** [comment]
closed by #148
